### ISSUE TITLE
Is it possible to convert just 1 pose (not the full sequence) from AMASS to HumanML3D ?

### ISSUE THREAD

**Issue by mayank64ce** [EricGuo5513/HumanML3D#98]

**Comment by mayank64ce** [comment]
What about SMPL to HumanML3D @EricGuo5513  ?

**Comment by jojo978** [comment]
I also found this issue. `process_file` in `motion_representation.ipynb` gives exceptions for sequences of the length 1. I think that's because it can't calculate the velocity.

**Comment by mayank64ce** [comment]
Yeah. Idk if I just pad the next frame with zeros, the output will make any sense.

**Comment by mayank64ce** [comment]
@jojo978  did you find any solution to this ?

**Comment by EricGuo5513** [comment]
I think for 1 pose, you can skip the velocity features,

**Comment by EricGuo5513** [comment]
Hello, these are the meaning of each indices for a HumanML3D feature vector:

# Recover global angle and positions for rotation data#
root_rot_velocity (B, seq_len, 1)# root_linear_velocity (B, seq_len,
2)# root_y (B, seq_len, 1)# ric_data (B, seq_len, (joint_num - 1)*3)#
rot_data (B, seq_len, (joint_num - 1)*6)# local_velocity (B, seq_len,
joint_num*3)# foot contact (B, seq_len, 4)

For a single pose, I guess you could have a dummy pose with z+ face
direction, as the first pose. And you are not able to get the foot
contact and local velocity. For the other features, they normally only
need the global rotation velocity, which in your case is the root
rotation of this single pose.
